README.md

```diff
@@ -19,7 +20,16 @@ be all set.
   - `PYPI_USERNAME` - The PyPI username to upload production packages
   - `PYPI_PASSWORD` - The PyPI password to upload production packages
 
-## Doof
+`bot_local.py` also requires these environment variables to be set, though environment
+variable checks may become more fine grained in the future. Until then it may be easiest
+to fill in fake values for environment variables not needed for your command.
+
+## Deployment
+Doof is a Slack bot running on heroku at `odl-release-bot`. New code will be
+deployed whenever a pull request is merged to `master`. Exceptions can be viewed
+in the heroku logs for `odl-release-bot`.
```

Comment:
I'd adjust this slightly to explain that deployments happen automatically and only after the tests pass on all of our environment branches and then expand on how those map (e.g. `master` -> `ci`, etc).

Reply:
oh ok